src/ui/components/search/autocompletecomponent.js

```diff
@@ -179,6 +186,15 @@ export default class AutoCompleteComponent extends Component {
       this._isOpen = true;
     }
 
+    if (this.customPrompts && !queryInputEl.value && data.sections) {
```

Comment:
Does this conditional mean that we don't show the custom prompts if there's some text in the `input`? What if the text in the input matches what's in some of the prompts?

Reply:
Yes. Product was ok with custom prompts disappearing when user start typing. Although, they also ask to match the behavior of the hardcoded prompts that user can configure in the backend, which does include the prompts if input have some sort of text matching. I can add that check using the levenshtein functions we used in headless